src/Invoice/Renderer/JsonRenderer.php

```diff
@@ -10,13 +10,16 @@
 namespace App\Invoice\Renderer;
 
 use App\Entity\InvoiceDocument;
+use App\Export\Base\DispositionInlineInterface;
+use App\Export\Base\DispositionInlineTrait;
 use App\Invoice\InvoiceFilename;
 use App\Invoice\InvoiceModel;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 
-final class JsonRenderer extends AbstractTwigRenderer
+final class JsonRenderer extends AbstractTwigRenderer implements DispositionInlineInterface
```

Comment:
I tried locally and JSON is not helping inline IMO. 
If someone uses JSON (some IT companies do for further processing) this is very likely via the InvoiceCommand. Same goes for TEXT invoice format.

Reply:
Yes, you're right. I realized that I've installed a json viewer extension in my chrome, therefore JSON is formatted pretty well. But this is not the standard.

Reply:
Further test here revealed: XSLX and DOCX are downloaded no matter which disposition is used.
So in my opinion it is enough to add the interface only to the PDF renderer.